frontend/catalyst/device/op_support.py

```diff
@@ -41,6 +41,28 @@ def is_supported(op: Operator, capabilities: DeviceCapabilities) -> bool:
     return op.name in capabilities.operations
 
 
+def is_lowering_compatible(op: Operator) -> bool:
+    """Check if an operation can be lowered to MLIR using JAX primitives."""
+    # Exceptions for operations that are not quantum instructions but are allowed
+    # via custom lowering rules.
+    # TODO: Revisit this as more explicit ops will be added to Catalyst Compiler.
+    if isinstance(op, (qml.Snapshot, qml.PCPhase, qml.MultiRZ)):
+        return True
+
+    # Accepted hyperparameters for quantum instructions bind calls
+    _accepted_hyperparams = {
```

Comment:
@albi3ro Here's the minimum set of hyperparams that we always need to check for compatibility with the `quantum.custom` MLIR Op.